gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc

```diff
@@ -0,0 +1,23 @@
+= Viewing Logs in Runtime Manager
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::flex-gateway-monitoring.adoc, flex-view-logs-in-monitoring.adoc
+:imagesdir: ../assets/images
+
+Use Runtime Manager to view the runtime and access logs of your Flex Gateway. Runtime Manager enables you search for key words within the logs and to filter by *Replica*, *Time Range*, and *Log Level*.
+
+NOTE: For an API's access logs to be visible, you must apply a Message Logging policy to that API. To learn more, see xref:flex-conn-message-log.adoc[].
+
+To access Flex Gateway logs in the Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the navigation menu.
+. Click the name of the gateway to which you want to add an API.
```

Comment:
I'm a little confused. Are they adding an API here? Or are they ultimately viewing logs in Runtime Manager?